spec/features/collection_previews/service_and_tool_tab_spec.rb

```diff
@@ -0,0 +1,127 @@
+describe 'Service and Tool Tabs', reset_provider: true do
```

Comment:
I think we might want tests for what shows up or doesn't when there are no associated services or tools